Source/EntityFramework.Extended/Batch/SqlServerBatchRunner.cs

```diff
@@ -223,11 +234,21 @@ private int InternalUpdate<TEntity>(ObjectContext objectContext, EntityMap entit
                 if (objectContext.CommandTimeout.HasValue)
                     updateCommand.CommandTimeout = objectContext.CommandTimeout.Value;
 
+                string tableName;
+                if (entityMap.SchemaName != null)
+                {
+                    tableName = new StringBuilder(50)
```

Comment:
This is not what StringBuilder is for... you use it when you have [multiple varying things](https://msdn.microsoft.com/en-us/library/ms228504.aspx)  to concatenate in a loop.

In this case, much better to simply do the concatenation by simple + operator which C# compiler will [compile as a String.Concat](http://stackoverflow.com/a/10341208/2076) call.